packages/mcp-server-supabase/vitest.config.ts

```diff
@@ -3,6 +3,6 @@ import { defineConfig } from 'vitest/config';
 export default defineConfig({
   test: {
     include: ['src/**/*.{test,spec}.ts'],
-    testTimeout: 20_000, // PGlite can take a while to initialize
+    testTimeout: 30_000, // PGlite can take a while to initialize
```

Comment:
Had a few tests that failed due to timeout the fixed upon re-run. Bumping this to 30s.